android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1691,6 +1795,71 @@ private WritableMap getAdInfo(final MaxAd ad)
         return adInfo;
     }
 
+    private WritableMap getWaterfallInfo(final MaxAdWaterfallInfo waterfallInfo)
+    {
+        WritableMap waterfall = Arguments.createMap();
+        waterfall.putString( "name", !TextUtils.isEmpty( waterfallInfo.getName() ) ? waterfallInfo.getName() : "" );
```

Comment:
No need to null-check where the getter is not marked as `@Nullable`.

Reply:
Fixed iOS as well.